packages/kbn-config-schema/src/byte_size_value/index.ts

```diff
@@ -35,9 +35,13 @@ export class ByteSizeValue {
   public static parse(text: string): ByteSizeValue {
     const match = /([1-9][0-9]*)(b|kb|mb|gb)/.exec(text);
     if (!match) {
-      throw new Error(
-        `could not parse byte size value [${text}]. Value must be a safe positive integer.`
-      );
+      const number = Number(text);
```

Comment:
__note:__ please add a note about coercion to the [`Readme` file](https://github.com/elastic/kibana/tree/master/packages/kbn-config-schema#schemabytesize) (similar to note we have for `number` already)